src/dodal/devices/pressure_jump_cell.py

```diff
@@ -0,0 +1,239 @@
+            self.pump_forward_limit = epics_signal_r(
+                PressureJumpCellLimitSwitch, prefix + "D74IN1"
+            )
+            self.pump_backward_limit = epics_signal_r(
+                PressureJumpCellLimitSwitch, prefix + "D74IN0"
+            )
+            self.pump_motor_direction = epics_signal_r(
+                PressureJumpCellPumpMotorDirection, prefix + "MTRDIR"
+            )
+            self.pump_speed_rbv = epics_signal_r(int, prefix + "MSPEED_RBV")
+
+            ## Pressure Transducer 1 ##
+            self.pressuretransducer1_omron_pressure = epics_signal_r(
+                float, prefix + "PP1:PRES"
+            )
+            self.pressuretransducer1_omron_voltage = epics_signal_r(
+                float, prefix + "PP1:RAW"
+            )
+            self.pressuretransducer1_beckhoff_pressure = epics_signal_r(
+                float, prefix + "STATP1:MeanValue_RBV"
+            )
+            self.pressuretransducer1_beckhoff_voltage = epics_signal_r(
+                float, adc2_prefix + "CH1"
+            )
+
+            ## Pressure Transducer 2 ##
+            self.pressuretransducer2_omron_pressure = epics_signal_r(
+                float, prefix + "PP2:PRES"
+            )
+            self.pressuretransducer2_omron_voltage = epics_signal_r(
+                float, prefix + "PP2:RAW"
+            )
+            self.pressuretransducer2_beckhoff_pressure = epics_signal_r(
+                float, prefix + "STATP2:MeanValue_RBV"
+            )
+            self.pressuretransducer2_beckhoff_voltage = epics_signal_r(
+                float, adc1_prefix + "CH2"
+            )
+
+            ## Pressure Transducer 3 ##
+            self.pressuretransducer3_omron_pressure = epics_signal_r(
+                float, prefix + "PP3:PRES"
+            )
+            self.pressuretransducer3_omron_voltage = epics_signal_r(
+                float, prefix + "PP3:RAW"
+            )
+            self.pressuretransducer3_beckhoff_pressure = epics_signal_r(
+                float, prefix + "STATP3:MeanValue_RBV"
+            )
+            self.pressuretransducer3_beckhoff_voltage = epics_signal_r(
+                float, adc1_prefix + "CH1"
+            )
+
+            ## Control Common ##
+            self.control_gotobusy = epics_signal_r(
+                PressureJumpCellBusyStatus, prefix + "CTRL:GOTOBUSY"
+            )
+
+            ## Control Pressure ##
+            self.control_timer = epics_signal_r(
+                PressureJumpCellTimerState, prefix + "CTRL:TIMER"
+            )
+            self.control_counter = epics_signal_r(float, prefix + "CTRL:COUNTER")
+            self.control_script_status = epics_signal_r(str, prefix + "CTRL:RESULT")
+            self.control_routine = epics_signal_r(str, prefix + "CTRL:METHOD")
+            self.control_state = epics_signal_r(str, prefix + "CTRL:STATE")
+            self.control_iteration = epics_signal_r(int, prefix + "CTRL:ITER")
```

Comment:
should: refactor this into more components such as is in for instance thawer.py https://github.com/DiamondLightSource/dodal/blob/main/src/dodal/devices/thawer.py

Reply:
those would be pressure transducer, valvecontrol, pressurecontroller,

Reply:
Improved as per https://github.com/DiamondLightSource/dodal/pull/673#discussion_r1691655374. There's now the following components: 
- PressureJumpCellControlValves
- PressureJumpCellPump
- PressureJumpCellPressureTransducers
- PressureJumpCellController

Reply:
Nit: I like this change but I would suggest that you don't need to prefix everything with `PressureJumpCell` it's implied by the fact it's in the `pressure_jump_cell` file